uavcan/file/407.Move.1.0.uavcan

```diff
@@ -0,0 +1,29 @@
+#
+# Manipulate a remote file system entry. Applies to files, directories, and links alike.
+# If the remote entry is a directory, all nested entries will be affected, too.
+#
+# Move/Copy
+#   Specify the source path and the destination path.
+#   If the source does not exist, the operation will fail.
+#   Set the preserve_source flag to copy rather than move.
+#   If the destination exists and overwrite_destination is not set, the operation will fail.
+#
+# Touch
+#   Specify the same path in the source and in the destination.
+#   Flags are ignored.
+#
+# Remove
```

Comment:
The type should probably be called "Modify" instead of Move. As it can both Move, Copy, Touch and Remove.

Reply:
Fixed